lighthouse-core/audits/installable-manifest.js

```diff
@@ -53,13 +53,13 @@ const UIStrings = {
   * @description Error message explaining that the manifest does not contain a suitable icon.
   * @example {192} value0
   */
-  'manifest-missing-suitable-icon': `Manifest does not contain a suitable icon - PNG, SVG or WebP format of at least {value0}\xa0px is required, the sizes attribute must be set, and the purpose attribute, if set, must include "any" or "maskable".`,
+  'manifest-missing-suitable-icon': `Manifest does not contain a suitable icon - PNG, SVG or WebP format of at least {value0}\xa0px is required, the sizes attribute must be set, and the purpose attribute, if set, must include "any" and should not include "maskable".`,
```

Comment:
I'm not sure this is true. It *must include* `any`, `maskable` should be optional added bonus.
```suggestion
  'manifest-missing-suitable-icon': `Manifest does not contain a suitable icon - PNG, SVG or WebP format of at least {value0}\xa0px is required, the sizes attribute must be set, and the purpose attribute, if set, must include "any".`,
```

https://w3c.github.io/manifest/#purpose-member

https://source.chromium.org/chromium/chromium/src/+/main:components/webapps/browser/installable/installable_manager.cc;l=159?q=DoesManifestContainRequiredIcon&ss=chromium

Reply:
@patrickhulce should we file upstream then? I copied this directly from the applications panel.

Reply:
Yeah I think so. Chromium is inconsistent [with itself](https://source.chromium.org/chromium/chromium/src/+/main:components/webapps/browser/installable/installable_logging.cc;l=32;bpv=1;bpt=1?q=%22Manifest%20does%20not%20contain%20a%20suitable%20icon%22&ss=chromium).